tfjs-backend-webgpu/src/backend_webgpu.ts

```diff
@@ -860,8 +908,13 @@ export class WebGPUBackend extends KernelBackend {
       this.commandQueueOwnedIds.add(input.dataId);
     });
     this.commandQueueOwnedIds.add(output.dataId);
-
-    if (env().get('WEBGPU_DEFERRED_SUBMIT_BATCH_SIZE') as
+    if (program.pixelsOpType === webgpu_program.PixelsOpType.TO_PIXELS) {
+      this.ensureComputePassEnded();
+      this.drawTextureToCanvas(
```

Comment:
How about moving `drawTextureToCanvas` to `ToPixels`?

Reply:
Currently all webgpu related code are kept inside backend_webgpu. So I prefer puting drawTextureToCanvas inside backend_webgpu. This also provide a opportunity to batch graphics and compute commands.

Reply:
Got it, thanks.